[CI] BaseInferenceActionTests testResolveTimeoutForTaskType_NonNullTimeout_ReturnsTimeout failing

**Build Scans:**
- [elasticsearch-intake #39124 / part5](https://gradle-enterprise.elastic.co/s/bquqlty5fiwza)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:inference:test" --tests "org.elasticsearch.xpack.inference.rest.BaseInferenceActionTests.testResolveTimeoutForTaskType_NonNullTimeout_ReturnsTimeout" -Dtests.seed=550D7C62A66EA5F7 -Dtests.locale=af-NA -Dtests.timezone=Europe/Athens -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.inference.rest.BaseInferenceActionTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testResolveTimeoutForTaskType_NonNullTimeout_ReturnsTimeout'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: is <0s>
     but: was <30s>
```

**Issue Reasons:**
- [main] 1 consecutive failures in test testResolveTimeoutForTaskType_NonNullTimeout_ReturnsTimeout
- [main] 1 consecutive failures in step part5
- [main] 1 consecutive failures in pipeline elasticsearch-intake

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -524,6 +524,3 @@ tests:
 - class: org.elasticsearch.index.mapper.DateRangeDocValuesLoaderTests
   method: testEmptyRangesAppendsNull
   issue: https://github.com/elastic/elasticsearch/issues/148378
-- class: org.elasticsearch.xpack.inference.rest.BaseInferenceActionTests
-  method: testResolveTimeoutForTaskType_NonNullTimeout_ReturnsTimeout
-  issue: https://github.com/elastic/elasticsearch/issues/148396

--- a/x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/rest/BaseInferenceActionTests.java
+++ b/x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/rest/BaseInferenceActionTests.java
@@ -249,7 +249,7 @@ public class BaseInferenceActionTests extends RestActionTestCase {
     }
 
     public void testResolveTimeoutForTaskType_NonNullTimeout_ReturnsTimeout() {
-        var timeout = randomTimeValue();
+        var timeout = randomTimeValue(1, 1000);
         for (TaskType taskType : TaskType.values()) {
             assertThat(resolveTimeoutForTaskType(taskType, timeout), is(timeout));
         }

